frontend/src/components/dashboard/DashboardTransactions.jsx

```diff
@@ -1,94 +1,140 @@
 import Link from "next/link";
 
-import React from "react";
+import React, { Component } from "react";
 
 import TransactionIcon from "../../../public/static/images/icon-t-transactions.svg";
 
 import { Row, Col } from "react-bootstrap";
 
 import Content from "../utils/Content";
-import TransactionsList from "../transactions/TransactionsList";
-
-export default ({ transactions }) => (
-  <Content
-    title={<h2>Recent Transactions</h2>}
-    icon={<TransactionIcon style={{ width: "22px" }} />}
-    size="medium"
-    border={false}
-    className="dashboard-transactions"
-  >
-    <Row className="px-0">
-      <Col xs="1" className="pr-0">
-        <div className="dashboard-blocks-hr-parent">
-          <div className="dashboard-blocks-hr" />
-        </div>
-      </Col>
-      <Col xs="11" className="px-0 dashboard-transactions-list">
-        <TransactionsList
-          transactions={transactions}
-          viewMode="compact"
-          reversed
-        />
-        <Row noGutters>
-          <Col xs="1" className="dashboard-transactions-icon-col" />
-          <Col xs="6">
-            <Link href="transactions">
-              <a className="dashboard-footer">View All</a>
-            </Link>
+import ActionsList from "../transactions/ActionsList";
+
+import TransactionsApi from "../../libraries/explorer-wamp/transactions";
+import FlipMove from "react-flip-move";
+
+export default class extends Component {
+  state = {
+    transactions: this.props.transactions
+  };
+
+  componentDidMount() {
+    this.regularFetchInfo();
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timer);
+    this.timer = false;
+  }
+
+  fetchInfo = async () => {
+    const transactions = await new TransactionsApi()
+      .getLatestTransactionsInfo()
+      .catch(() => null);
+    this.setState({ transactions });
+  };
+
+  regularFetchInfo = async () => {
+    await this.fetchInfo();
+    if (this.timer !== false) {
+      this.timer = setTimeout(this.regularFetchInfo, 10000);
+    }
+  };
+
+  render() {
+    const { transactions } = this.state;
+    let reserved = true;
+    let actions = transactions.map(transaction => (
+      <ActionsList
+        key={transaction.hash}
+        actions={transaction.actions}
+        transaction={transaction}
+        viewMode="compact"
+        reversed={true}
+      />
+    ));
+
+    if (reserved) actions.reverse();
+
+    return (
+      <Content
+        title={<h2>Recent Transactions</h2>}
+        icon={<TransactionIcon style={{ width: "22px" }} />}
+        size="medium"
+        border={false}
+        className="dashboard-transactions"
+      >
+        <Row className="px-0">
+          <Col xs="1" className="pr-0">
+            <div className="dashboard-blocks-hr-parent">
+              <div className="dashboard-blocks-hr" />
+            </div>
+          </Col>
+          <Col xs="11" className="px-0 dashboard-transactions-list">
+            {/* <FlipMove duration={1000} staggerDurationBy={0}> */}
```

Comment:
Is it intended to be commented out here?

Reply:
the transaction maybe have issues to do with FlipMove, because it is a double map to list the array, and it will cause an error, so I am still fix it